tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -727,9 +727,10 @@ not null when string.Equals(value, "otlp", StringComparison.OrdinalIgnoreCase) =
             // We create a lazy here because this is kind of expensive, and we want to avoid calling it if we can
             _fallbackApplicationName = new(() => ApplicationNameHelpers.GetFallbackApplicationName(this));
 
-            InitialMutableSettings = MutableSettings.CreateInitialMutableSettings(source, telemetry, errorLog, this);
-            MutableSettings = InitialMutableSettings;
-            Manager = new(this, InitialMutableSettings, Exporter);
+            // Move the creation of these settings inside SettingsManager?
+            var initialMutableSettings = MutableSettings.CreateInitialMutableSettings(source, telemetry, errorLog, this);
+            Manager = new(this, initialMutableSettings, Exporter);
+            MutableSettings = initialMutableSettings;
```

Comment:
Was the order of operations here changed on purpose? `Manager`  / `MutableSettings` got flipped around but I don't see anything really so just wondering